config/environments/development.rb

```diff
@@ -72,4 +72,18 @@
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = {host: ENV["HOST_NAME"] }
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+    :user_name => ENV["NAME"],
```

Comment:
fix tương tự bên dưới
```suggestion
    user_name: ENV["NAME"],
```